docs/introduction/01-install.rst

```diff
@@ -38,24 +38,24 @@ Run the demo project in docker (Step 2)
 ****************************************
 
 Info: The `demo project <https://github.com/django-cms/django-cms-quickstart>`_ is a minimal
-django project with some additional requirements in the requirements.txt.
+django project with some additional requirements in the ``requirements.txt``.
 
-Open the terminal application on your computer and go to a safe folder (i.e. cd ~/Projects), then:
+Open the terminal application on your computer and go to a safe folder (i.e. ``cd ~/Projects``), then:
 
 
 ::
 
       git clone https://github.com/django-cms/django-cms-quickstart.git
       cd django-cms-quickstart
-      docker compose build web
-      docker compose up -d database_default
-      docker compose run web python manage.py migrate
-      docker compose run web python manage.py createsuperuser
-      docker compose up -d
+      docker-compose build web
+      docker-compose up -d database_default
+      docker-compose run web python manage.py migrate
+      docker-compose run web python manage.py createsuperuser
+      docker-compose up -d
```

Comment:
This is the old compose version.

It used to be a third party package but has been brought into docker cli so it's `docker compose` these days.

This changed in compose v2 sometime last year.

Docs here: https://docs.docker.com/compose/#compose-v2-and-the-new-docker-compose-command

Reply:
Hi Mark, 

Thank you for letting me know. I now know I was using a Docker version lower V2. I'll revert the changes done to these commands by getting rid of the dashes.

Reply:
@luisnavarrodelangel Yeah it might be worth a note that the command changed. The `versionadded` directive might work for this.
https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionadded